src/App.tsx

```diff
@@ -15,36 +15,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabet = 'Alphabet',
+  Length = 'Length',
+  Default = '',
+}
+
+function prepareGoods(goods: string[], type: SortType, reverse: boolean) {
+  const preparedGoods = [...goods];
+
+  preparedGoods.sort((good1, good2) => {
```

Comment:
1. move this logic higher
` const compareGoods = (a: string, b: string) => {...}`
2. const sortedGoods = [...goods].sort(compareGoods);